metadataStore/commands.py

```diff
@@ -276,30 +241,34 @@ def find_header(limit=50, **kwargs):
         pass
 
     try:
-        st_time_dict = kwargs.pop('create_time')
-        if not isinstance(st_time_dict, dict):
-            raise ValueError('Wrong format. create_time must include '
+        time_dict = kwargs.pop('time')
+        if not isinstance(time_dict, dict):
```

Comment:
Yup, I have to do this for all time related searches. There are quite a few of those. I think I need to make this a routine and call it instead of typing over and over